### ISSUE TITLE
UnquotedVariable: false positive for salt

### ISSUE THREAD

**Issue by arthurzam** [pkgcore/pkgcheck#490]
Example code which flags TMPDIR:

```bash
python_test() {
	# testsuite likes lots of files
	ulimit -n 4096 || die

	# ${T} is too long a path for the tests to work
	local TMPDIR
	TMPDIR="$(mktemp --directory --tmpdir=/tmp ${PN}-XXXX)"
	(
		export TMPDIR
		cleanup() { rm -rf "${TMPDIR}" || die; }

		trap cleanup EXIT

		addwrite "${TMPDIR}"

		USE_SETUPTOOLS=1 NO_INTERNET=1 SHELL="/bin/bash" \
			"${EPYTHON}" -m pytest -vv \
			|| die "testing failed with ${EPYTHON}"
	)
}
```

https://gitweb.gentoo.org/repo/gentoo.git/tree/app-admin/salt/salt-3005.ebuild#n191
